transports/websocket/src/lib.rs

```diff
@@ -315,20 +313,20 @@ mod tests {
 
     use super::Config;
 
-    #[test]
-    fn dialer_connects_to_listener_ipv4() {
+    #[tokio::test]
+    async fn dialer_connects_to_listener_ipv4() {
         let a = "/ip4/127.0.0.1/tcp/0/ws".parse().unwrap();
-        futures::executor::block_on(connect(a))
+        connect(a).await
     }
 
-    #[test]
-    fn dialer_connects_to_listener_ipv6() {
+    #[tokio::test]
+    async fn dialer_connects_to_listener_ipv6() {
         let a = "/ip6/::1/tcp/0/ws".parse().unwrap();
-        futures::executor::block_on(connect(a))
+        connect(a).await
     }
 
-    fn new_ws_config() -> Config<tcp::async_io::Transport> {
-        Config::new(tcp::async_io::Transport::new(tcp::Config::default()))
+    fn new_ws_config() -> Config<tcp::tokio::Transport> {
+        Config::new(tcp::tokio::Transport::new(tcp::Config::default()))
     }
```

Comment:
Same here. This should be a separate PR, and the current PR should only be merged afterwards.